modules/alpha/kustomization.yaml

```diff
@@ -1,3 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
 
+resources:
+- moduletemplate-keda.yaml
```

Comment:
not sure if it will have an effect but the target file is called `moduletemplate-keda..yaml` with en extra dot before ext.

Reply:
will check it out, thx

Reply:
fixed